python/interpret_community/explanation/explanation.py

```diff
@@ -220,17 +220,20 @@ class FeatureImportanceExplanation(BaseExplanation):
     :type features: Union[list[str], list[int]]
     """
 
-    def __init__(self, features=None, is_raw=False, **kwargs):
+    def __init__(self, features=None, is_raw=False, is_eng=False, **kwargs):
```

Comment:
not sure if we should change it to is_engineered or not... something to think about

Reply:
changed most of them